src/pages/Event/utils/handleEventQuery.ts

```diff
@@ -0,0 +1,26 @@
+import { getEventById } from './../../../actions/EventsActions';
```

Comment:
I notice src/utils works, what about src/assets?

Reply:
I'm not sure if I fully understand the question, can you give me more details please?

Reply:
Below on line 4 I noticed this:
![image](https://user-images.githubusercontent.com/36519478/174304388-1e18f641-086c-4d63-aee0-0ddc63ecc52a.png)

Seems like you are importing something from `src/utils`, but in line 1 you have to move 4 levels up to reach src, I was wondering if you can use `src/actions/EventsAction` as you did on line 4

Reply:
Oh, great!, thanks for clarify. I will test if it works

Reply:
Looks better

![image](https://user-images.githubusercontent.com/66505715/174409029-775e2977-0df7-4c3a-a8d7-bbc7e5a49fa5.png)

Thanks!